src/components/dialogs/create-study-dialog.js

```diff
@@ -238,6 +238,15 @@ export const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
                         p.possibleValues = p.possibleValues?.sort((a, b) =>
                             a.localeCompare(b)
                         );
+                        // we check if the param is for extension, if it is, we select all possible values by default.
+                        // the only way for the moment to check if the param is for extension, is by checking his name.
+                        //TODO to be removed when extensions param default value corrected in backend to include all possible values
+                        if (
+                            p.type === 'STRING_LIST' &&
+                            p?.name?.endsWith('extensions')
```

Comment:
can you rename p to parameter please?
you can create a const for 'STRING_LIST'
p?: ? is unnecessary